sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/query.sql

```diff
@@ -374,7 +374,7 @@ main_ping AS (
     ARRAY_AGG(normalized_os_version RESPECT NULLS ORDER BY submission_date)[
       SAFE_OFFSET(0)
     ] AS normalized_os_version,
-    ARRAY_AGG(startup_profile_selection_reason_first_ping_only RESPECT NULLS ORDER BY submission_date)[
+    ARRAY_AGG(startup_profile_selection_first_ping_only RESPECT NULLS ORDER BY submission_date)[
       SAFE_OFFSET(0)
     ] AS startup_profile_selection_reason,
```

Comment:
Does this `startup_profile_selection_reason` column name still make sense given the upstream column name is different?

Reply:
Not perfectly (the upstream was also a typo 🤦) but the column description does clarify somewhat